SilverSolver/assignment3/word_search.py

```diff
@@ -0,0 +1,49 @@
+from bisect import bisect_left
+from dictionary import Dictionary
+from typing import List, Tuple
+
+def empty_matrix(m: int, n: int, value: bool) -> List[List[bool]]:
```

Comment:
You can write this in one line: 
  return [[value]*n for _ in range(m)]

From there, maybe you just inline this expression, for concision, if you think it is clearer.